vulture_os/toolkit/api_parser/symantec/symantec.py

```diff
@@ -64,8 +64,11 @@ def __init__(self, data):
 
     def test(self):
         try:
-            startDate = timezone.now()
-            startDate = startDate.replace(minute=0, second=0, microsecond=0)
+            startDate = datetime.datetime.now(timezone.utc) - datetime.timedelta(days=1)
```

Comment:
will that work if the fix is specifically avoiding getting long ranges of data?
shouldn't that be a 10 minutes interval instead, and shouldn't that be enough ?